gsoc/2019/mentors.md

```diff
@@ -35,6 +35,7 @@ layout: plain
 * Daniel Hynds [daniel.hynds@cern.ch](mailto:daniel.hynds@cern.ch) Nikhef
 * Tomas Javurek [tomas.javurek@cern.ch](mailto:tomas.javurek@cern.ch) CERN
 * Maarten de Jong [m.de.jong@nikhef.nl](mailto:m.de.jong@nikhef.nl) Nikhef
+* Moritz Kiehn [moritz.kiehn@cern.ch](mailto:moritz.kiehn@cern.ch) Université de Genève
```

Comment:
Could you please add an organization description for the *Université de Genève* ?